miqi/runtime/app_server.py

```
@@ -848,7 +848,26 @@ async def _chat_abort(request_id, params, client_id, session_id, registry):
         # explicit thread registers its cancel event under session_key, so an
         # abort that resolved to "default" could never signal it (#542).
         thread_id = typed.thread_id or params.get("session_key") or "default"
-        await session.submit(AbortTurn(thread_id=thread_id))
+        try:
+            await session.submit(AbortTurn(thread_id=thread_id))
+        except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)